src/main/resources/db/migrations/V001__CreateSchemaTables.sql

```diff
@@ -0,0 +1,171 @@
+-------------------------------------- Create types
+CREATE TYPE public.COURT_TYPE AS ENUM ('crown', 'magistrate', 'family');
+CREATE TYPE public.PARTICIPANT_TYPE AS ENUM ('witness','defendant');
+CREATE TYPE public.RECORDING_STATUS AS ENUM ('standby', 'initialisation', 'recording', 'finished', 'processing', 'available', 'future');
+CREATE TYPE public.RECORDING_ORIGIN AS ENUM ('pre', 'vodafone');
+CREATE TYPE public.AUDIT_LOG_SOURCE AS ENUM ('application', 'portal', 'admin', 'auto');
+CREATE TYPE public.AUDIT_LOG_TYPE AS ENUM ('create', 'update', 'delete', 'action');
+
+-------------------------------------- Create tables
+CREATE TABLE public.regions (
+    id UUID PRIMARY KEY NOT NULL,
+    name VARCHAR(100)
+);
+
+CREATE TABLE public.courts (
+    id UUID PRIMARY KEY NOT NULL,
+    court_type COURT_TYPE NOT NULL,
+    name VARCHAR(255) NOT NULL,
+    location_code VARCHAR(25)
+);
+
+CREATE TABLE public.rooms (
+    id UUID PRIMARY KEY NOT NULL,
+    room VARCHAR(45) NOT NULL
+);
+
+CREATE TABLE public.court_regions(
+    id UUID PRIMARY KEY NOT NULL,
+    court_id UUID REFERENCES courts(id) NOT NULL,
+    region_id UUID REFERENCES regions(id) NOT NULL
+
+);
+
+CREATE TABLE public.courtrooms (
+    id UUID PRIMARY KEY NOT NULL,
+    court_id UUID REFERENCES courts(id) NOT NULL,
+    room_id UUID REFERENCES rooms(id) NOT NULL
+);
+
+CREATE TABLE public.cases (
+    id UUID PRIMARY KEY NOT NULL,
+    court_id UUID REFERENCES courts(id) NOT NULL ,
+    case_ref VARCHAR(25) NOT NULL, 
+    test BOOLEAN DEFAULT FALSE,
+    deleted BOOLEAN DEFAULT FALSE,
+    created_on TIMESTAMPTZ NOT NULL DEFAULT NOW(),
+    modified_on TIMESTAMPTZ DEFAULT NOW()
+);
+
+
+CREATE TABLE public.participants (
+    id UUID PRIMARY KEY NOT NULL,
+    case_id UUID REFERENCES cases(id) NOT NULL,
+    participant_type PARTICIPANT_TYPE NOT NULL,
+    first_name VARCHAR(100) NOT NULL,
+    last_name VARCHAR(100) NOT NULL,
+    deleted BOOLEAN DEFAULT FALSE,
+    created_on TIMESTAMPTZ NOT NULL DEFAULT NOW(),
+    modified_on TIMESTAMPTZ DEFAULT NOW()
+);
+
+CREATE TABLE public.bookings (
+    id UUID PRIMARY KEY NOT NULL,
+    case_id UUID REFERENCES cases(id) NOT NULL,
+    date DATE NOT NULL,
+    deleted BOOLEAN DEFAULT FALSE,
+    created_on TIMESTAMPTZ NOT NULL DEFAULT NOW(),
+    modified_on TIMESTAMPTZ DEFAULT NOW()
+);
+
+
+CREATE TABLE public.booking_participants (
+    id UUID PRIMARY KEY NOT NULL,
+    participant_id UUID REFERENCES participants(id) NOT NULL,
+    booking_id UUID REFERENCES bookings(id) NOT NULL
+);
+
+CREATE TABLE public.users (
+    id UUID PRIMARY KEY NOT NULL,
+    first_name VARCHAR(100) NOT NULL,
+    last_name VARCHAR(100) NOT NULL,
+    email VARCHAR(100) NOT NULL,
+    organisation VARCHAR(250),
```

Comment:
did we decide that organisations shouldn't get their own table?

Reply:
I think the idea is to leave with users for now, since we're using this table to track both internal and external users. We'll be getting the organisation details from Azure B2C identities, but from my understanding we'll only be using this field for reporting purposes, so the plan at the moment, is to keep it all in users. Let me know if you suggest different though